tests/core/test_artifact.py

```diff
@@ -280,12 +276,42 @@ def test_create_from_dataframe(df):
     assert artifact.n_observations == 2
     assert hasattr(artifact, "_local_filepath")
     artifact.save()
-    # can do backed now, tested in test_storage.py
```

Comment:
@Koncopd, I'm not sure if you or @sunnyosun added it -- the downstream streaming functionality shouldn't appear in `test_artifact.py` and given the comment says it's already tested under "storage" it seems save to remove it here. Agreed?

Reply:
Why shouldn't it appear? It is artifact functionality. I would prefer to keep it here instead of deleting.

Reply:
But yes, it looks like this is tested also in `test_storage.py`.

Reply:
I mean i am fine if you delete this, but i also don't get why not keep it here.

Reply:
> But yes, it looks like this is tested also in test_storage.py.

Great!

> I mean i am fine if you delete this

Then I'll delete it and we can restore a version of this after thinking through whether it's actually needed given test_storage.py takes care of testing it